test/unit/math/rev/prob/neg_binomial_2_ccdf_log_test.cpp

```diff
@@ -18,7 +18,7 @@ TEST(ProbDistributionsNegBinomial2, derivatives_lccdf) {
   std::vector<double> gradients;
   val.grad(x, gradients);
 
-  double epsilon = 1e-6;
+  double epsilon = 1e-5;
```

Comment:
If this worked before, why doesn't it work now? `1e-6` now is less stringent than previously.

Reply:
The epsilon here is the interval the discrete diff is calculated over.  This is a very flat region of the function so I don't think it's necessarily less stringent.  I am concerned about this being wrong and I am would prefer to include a different test of the negative binomial that didn't get into the discrete diff.  I can probably get a good reference value from R or mathematica.

Reply:
Oh lol, yes, this is the finite difference.

Reply:
Turns out that with the other fixes I can revert this change.  It's still bizarre.